src/main/java/com/litografiaartesplanchas/inventoryservice/controller/MaterialController.java

```diff
@@ -13,12 +13,12 @@
 
 
 @RestController
-@RequestMapping("/api/inventory")
+@RequestMapping("/api/v1/inventory")
 public class MaterialController {
     @Autowired
     private MaterialService materialService;
 
-    @PostMapping
+    @PostMapping("/save")
     public ResponseEntity<Object> saveMaterial(@RequestBody Material material) {
         try {
             materialService.saveMaterial(material);
```

Comment:
Manage the response of repository. If return false throw some error to response with status 400